packages/components/src/interfaces/complex/LogoCloud.ts

```diff
@@ -0,0 +1,39 @@
+import type { Static } from "@sinclair/typebox"
+import { Type } from "@sinclair/typebox"
+
+export const LogoCloudSchema = Type.Object(
+  {
+    type: Type.Literal("logocloud", { default: "logocloud" }),
+    images: Type.Array(
+      Type.Object({
+        src: Type.String({
+          title: "Upload image",
+          format: "image",
+        }),
+        alt: Type.String({
+          title: "Alternate text",
+          maxLength: 120,
+          description:
+            "Add a descriptive alternative text for this image. This helps visually impaired users to understand your image.",
+        }),
+      }),
```

Comment:
note to self @adriangohjw - to update this to use the common imagealt and imagesrc schema after https://github.com/opengovsg/isomer/pull/920 is merged